python/test/unit/test_dummy_not_landing.py

```diff
@@ -0,0 +1,75 @@
+import triton
+import triton.language as tl
+import torch
+
+
+@triton.jit
+def k_mean(X, Mean, Var, stride, N, **META):
+    # fmt: on
+    """
+    Fused layernorm kernel over a 3d tensor.
+    The layer norm is applied over the last dimension.
+
+    Compute
+        y = (x - E(x))/(sqrt(var(x) + epsilon)) * gamma + beta
+    """
+
+    row = tl.program_id(0)
+    cols = tl.arange(0, META["BLOCK_SIZE_N"])
+
+    # Move to this row
+    x_ptrs = X + row * stride + cols
+    x = tl.load(x_ptrs, mask=cols < N, other=0.0).to(tl.float32)
+    x = tl.where(cols < N, x, 0.0)
+
+    # Compute variance
+    x_mean = tl.sum(x, axis=0) / N
+    x_zm = x - x_mean
+    x_zm = tl.where(cols < N, x_zm, 0.0)  # THIS SHOULD NOT BE NEEDED
```

Comment:
unless I'm really missing something @ptillet , x and x_mean should be zero here beyond N, so this should do nothing. If you check out the test, for a non-power-of-two tensor size, it changes the result a fair bit (you need that to get something correct)